msi/release.sh

```diff
@@ -177,6 +177,9 @@ BUILD_EXTRA_WINPATH="$(cd "$SCRIPT_PATH"/.. && pwd -W | tr / \\\\)"
             <Component Directory="INSTALLFOLDER:\\usr\\share\\git\\">
                 <File Id="ReleaseNotes_Css" Source="$BUILD_EXTRA_WINPATH\\usr\\share\\git\\ReleaseNotes.css" />
             </Component>
+            <Component Directory="INSTALLFOLDER">
+                <RegistryValue Root="HKMU" Key="Software\GitForWindows" Name="CurrentVersion" Type="string" Value="\$(var.Version)" />

```

Comment:
Ahh, I missed that this is in release.sh. Why is this change i release.sh? releases.sh should have the minimal necessary to translate the text file with all the file paths into Component elements. Everything else can live much more readable in .wxs files. Right?